compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -772,6 +772,16 @@ public void visit(BLangPackage pkgNode) {
                 initFnBody.stmts.add(constInit);
             }
         }
+        HashSet<String> elements = new HashSet<String>();
```

Comment:
Shall we extract this code into a method with a descriptive name?

Reply:
Okay. I'll change it.